Simulator/game_round.py

```diff
@@ -321,6 +292,68 @@ def carousel8_4(self):
                 player.add_to_bench(ran_cost_5)
                 player.refill_item_pool()
 
+    def decide_player_combat(self, players):
+        player_list = []
+        self.matchups = []
+        for player in players.values():
+            if player:
+                player_list.append(player.player_num)
+        random.shuffle(player_list)

```

Comment:
very small thing, you could add an if statement prior to this saying to do this only if there are an odd number of players. Small (and I mean really small) time optimization.